src/MainPage.js

```diff
@@ -2,33 +2,40 @@ import { useEffect } from 'react'
 import styled from 'styled-components/macro'
 import Form from './components/Form'
 
-
-function MainPage({setUserPlace, errorMessage, isCountyDataLoaded, resetSearch}) {   
-    
-    useEffect(() => {isCountyDataLoaded && resetSearch()}, [])
+function MainPage({
+    setUserPlace,
+    errorMessage,
+    isDataLoading,
+    isCountyDataLoaded,
+    resetSearch,
+    setSearchOrigin,
+}) {
+    useEffect(() => {
+        isCountyDataLoaded && resetSearch()
+    }, [])
 
     return (
         <MainPageStyled>
             <h1>Bin ich in einem Covid-19 Hotspot?</h1>
-            <Form 
+            <Form
                 setUserPlace={setUserPlace}
                 errorMessage={errorMessage}
-                >
-            </Form>
-        </MainPageStyled> 
+                isDataLoading={isDataLoading}
+                setSearchOrigin={setSearchOrigin}
+            ></Form>
+        </MainPageStyled>
     )
 }
 
 const MainPageStyled = styled.div`
-padding: 10px;
-display: grid;
-grid-gap: 20px;
-background: #F5F5F7;
+    padding: 10px;
+    display: grid;
+    grid-gap: 20px;
+    background: #f5f5f7;
```

Comment:
Sometimes you work with color variables. So it's not quite stringent.

Reply:
thanks! Got the variables after I styled that one, I guess